aspnetcore/razor-pages/index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml.cs

```diff
@@ -36,10 +36,10 @@ public async Task<IActionResult> OnPostAsync()
             _context.Customers.Add(Customer);
             await _context.SaveChangesAsync();
 
-            return RedirectToPage("./Index");
+            return RedirectToPage("/Index");
```

Comment:
I don't think this is right. This redirects to the root index, you want to redirect to the Customers Index.

Reply:
@Rick-Anderson I have known.

Reply:
@Rick-Anderson I have corrected the error commit, other commits i think is no problem. Please review other commits and help to merge it.